crates/p2p_proto/build.rs

```diff
@@ -0,0 +1,13 @@
+use std::io::Result;
+
+fn main() -> Result<()> {
+    prost_build::compile_protos(
```

Comment:
Why not go with `rust-protobuf` or `quick-protobuf`? Then google's protoc won't be necessary as a hard runtime environment dependency in production build and in test fixtures. The libp2p seems to migrate to the latter: https://github.com/libp2p/rust-libp2p/issues/3024.

Reply:
Good to know, thanks! I'll look into using `quick-protobuf` instead.

Reply:
I'd propose we leave this as is for now -- until libp2p migrates to using `quick-protobuf`. Even if we get rid of `prost` now we'd still need `protoc` because of our libp2p dependency.

Once libp2p migrates we can migrate, too.